File: 1.Python/Program8.py
# 6.Write a program which accept number from user and check whether that number
# is positive or negative or zero.
# Input : 11
# Output : Positive Number
# Input : -8
# Output : Negative Number
# Input : 0
# Output : Zero
def main():
   No1=0
   print("Enter number to check:")
   No1=int(input())
   if No1==0:
       print("Zero")
   elif No1>0:
       print("Positive Number")
   else:
       print("Negative Number")
if __name__=="__main__":
   main()

File: 0.C_CPP_JAVA Programs/JAVA/Program8.java
/*3.Write a program which accept number from user and return the count of digits in between 3 and 7.
Input :  2395  Output :  1
Input :  1018 Output :  0
Input :  4521 Output :  2
Input :  9922 Output :  0
 */
import java.lang.*;
import java.util.*;

class Number
{
    int Count(int iNo)
    {
        if(iNo<0)
        {
            iNo=-iNo;
        }
        int iCount=0;
        int iDigit=0;
        while (iNo!=0)
        {
            iDigit=iNo%10;
            if(iDigit>3&&iDigit<7)
            {
                iCount++;
            }
            iNo=iNo/10;
        }
        return iCount;
    }
}
class Demo
{
    public static void main(String arg[])
    {
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the number:");
        int iValue=sobj.nextInt();

        Number nobj=new Number();
       int iRet=nobj.Count(iValue);
       System.out.println("Your Count of digit is: " + iRet);
    }
}